linalg/densemat.hpp

```diff
@@ -961,6 +961,12 @@ class DenseMatrixSVD
    DenseMatrixSVD(int h, int w,
                   bool left_singular_vectors=false,
                   bool right_singlular_vectors=false);
+   DenseMatrixSVD(DenseMatrix &M,
+                  char left_singular_vectors='A',
```

Comment:
Can you please add some doxygen comments about the different options or point to the relevant parts of the LAPACK documentation?

Reply:
Added the requested comments